pom.xml

```diff
@@ -378,6 +378,7 @@
             <configuration>
               <executable>${jvm.executable}</executable>
               <arguments>
+                <argument>-Des.path.home=.</argument>
```

Comment:
@dakrone is it ok if its a little slower but more "correct"? I'd really like for our "exec" method to use our real package and run bin/elasticsearch if possible. 

I think there is really good value to "just run the damn thing based on the current code", I would start using such a task immediately, because it would save me the steps of mvn package -DskipTests, cd target..., unzip, cd elastic-... bin/elasticsearch